src/app/mapController.js

```diff
@@ -177,6 +190,33 @@ define([
             graphic.getDojoShape().moveToFront();
 
             return graphic;
+        },
+        addLayers: function (layers) {
+            // summary:
+            //      toggles layers in the map
+            // layers: { graphicsLayers: [], dynamicLayers: [] }
+            console.log('app/mapController::addLayers', arguments);
+
+            var loaded = false;
+            layers.graphicsLayers.forEach(function (layer) {
+                loaded = this.map.graphicsLayerIds.some(function (id) {
+                    return id === layer.id;
+                }, this);
+
+                if (!loaded) {
+                    this.map.addLayer(layer);
+                }
+            }, this);
+
+            layers.graphicsLayers.forEach(function (layer) {
```

Comment:
This looks like a duplicate of 201. Was it intended to be `dynamicLayers` or something?